testing-pre-releases.md

```diff
@@ -0,0 +1,530 @@
+        - [Getting docker images from a GCR registry](#getting-docker-images-from-a-gcr-registry)
+    - [Create a local version](#create-a-local-version)
+        - [Build .debs packages](#build-debs-packages)
+        - [Build kubeadm binary](#build-kubeadm-binary)
+        - [Build controlplane docker images](#build-controlplane-docker-images)
+    - [Creating the Kubernetes cluster with kubeadm](#creating-the-kubernetes-cluster-with-kubeadm)
+    - [Testing the Kubernetes cluster](#testing-the-kubernetes-cluster)
+        - [Smoke test](#smoke-test)
+            - [DNS](#dns)
+            - [Deployments](#deployments)
+            - [Port Forwarding](#port-forwarding)
+            - [Logs](#logs)
+            - [Exec](#exec)
+            - [Services](#services)
+        - [Conformance test](#conformance-test)
+    - [Tips and Tricks](#tips-and-tricks)
+        - [Semantic version ordering](#semantic-version-ordering)
+        - [How to identify exact version/build number for a PR](#how-to-identify-exact-versionbuild-number-for-a-pr)
+        - [Change the target version number when building a local release](#change-the-target-version-number-when-building-a-local-release)
+
+<!-- /TOC -->
+
+## Kubeadm version skew policy
+
+The [Kubeadm version skew policy](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#version-skew-policy)
+can briefly be summarized in:
+
+- The kubeadm CLI tool of version MAJOR.MINOR may deploy clusters with a control plane of version MAJOR.MINOR or MAJOR.(MINOR-1).
+- The kubeadm CLI MAJOR.MINOR can upgrade an existing kubeadm-created cluster of version MAJOR.(MINOR-1).
+- One minor version skew between the kubelet and the control plane is supported, but the kubelet version may never
+  exceed the API server version.
+
+## Availability of pre-compiled release artifacts
+
+The availability of pre-compiled deb/yum packages, kubeadm binary file or controlplane docker images depends by the Kubernetes build and deploy infrastructure.
+
+The table below summarize the current state:
+
+|                         | .deb or .rpm                                                 | kubeadm binary                                               | control plane                                                |
+| ----------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **GA release**          | from .deb or .rpm repository                                 | from `gs://kubernetes-release/release/` GCS bucket | from `gcr.io/google-containers` container registry           |
+| **alpha/beta release*** | not available. use CI/CD version "near" tag                  | from `gs://kubernetes-release/release/` GCS bucket | from `gcr.io/google-containers` container registry           |
```

Comment:
Instead of mentioning the gs bucket for GA/alpha/beta releases, should we just mention the github release instead?